### PR TITLE
Feature/integration robustification 2

### PR DESCRIPTION
**Pull request by cstryker**
This is a big 'un. Addresses mainly:

https://issues.newrelic.com/browse/NR-368
https://issues.newrelic.com/browse/NR-370
https://issues.newrelic.com/browse/NR-389
https://issues.newrelic.com/browse/NR-365
https://issues.newrelic.com/browse/NR-367
https://issues.newrelic.com/browse/NR-369
https://issues.newrelic.com/browse/NR-435
https://issues.newrelic.com/browse/NR-439

### CODE REVIEW

**Comment by cstryker** [comment]
I'll resolve those merge requests once the PR is approved.

**Review by dsellarsnr — APPROVED** [review]

File: shared/agent/src/providers/clubhouse.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
 	FetchThirdPartyCardsRequest,
 	FetchThirdPartyCardsResponse,
 	MoveThirdPartyCardRequest,
+	ThirdPartyDisconnect,
 	ThirdPartyProviderCard
 } from "../protocol/agent.protocol";
 import { CSClubhouseProviderInfo } from "../protocol/api.protocol";
```
Comment (bcanzanella) [inline shared/agent/src/providers/clubhouse.ts:21]: thoughts on renaming these classes? Clubhouse is now known as Shortcut (https://shortcut.com/)

Comment (cstryker) [reply shared/agent/src/providers/clubhouse.ts:21]: There is another ticket (for Dillon) to rename Clubhouse -> Shortcut. Won't make it into this PR.

File: shared/ui/Stream/CrossPostIssueControls/IssueDropdown.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from "react";
```
Comment (bcanzanella) [inline shared/ui/Stream/CrossPostIssueControls/IssueDropdown.tsx:1]: this file was renamed to IssuePane.tsx

**Review by eric-j — APPROVED** [review]

**Pull request merged**